tests/index.test.ts

```diff
@@ -5,6 +5,7 @@ import { conventionalCommitsTypes } from '../src/index';
 describe('conventionalCommitsTypes() function test', () => {
   it.each([
     ['en.json', undefined],
+    ['en.json', 'unknown-locale'],
```

Comment:
```suggestion
    ['en.json', ''],
    ['en.json', 'unknown-locale'],
```

Add test for the empty string input.